packages/discord.js/src/structures/BaseGuildVoiceChannel.js

```diff
@@ -80,6 +132,45 @@ class BaseGuildVoiceChannel extends GuildChannel {
     );
   }
 
+  /**
+   * Creates an invite to this guild channel.
+   * @param {InviteCreateOptions} [options={}] The options for creating the invite
+   * @returns {Promise<Invite>}
+   * @example
+   * // Create an invite to a channel
+   * channel.createInvite()
+   *   .then(invite => console.log(`Created an invite with a code of ${invite.code}`))
+   *   .catch(console.error);
+   */
+  createInvite(options) {
+    return this.guild.invites.create(this.id, options);
+  }
+
+  /**
+   * Fetches a collection of invites to this guild channel.
+   * Resolves with a collection mapping invites by their codes.
+   * @param {boolean} [cache=true] Whether or not to cache the fetched invites
```

Comment:
Be rid of this superfluous wording!

```suggestion
   * @param {boolean} [cache=true] Whether to cache the fetched invites
```